app/cmd/post.go

```diff
@@ -0,0 +1,122 @@
+package cmd
+
+import (
+	"fmt"
+	"gopkg.in/russross/blackfriday.v2"
+	"html/template"
+	"io/ioutil"
+	"log"
+	"os"
+	"path"
+	"path/filepath"
+	"strconv"
+	"strings"
+	"time"
+)
+
+func NewPost(filename string) Post {
+	// Get filename without extension
+	filenameBase := fNameWithoutExtension(filename)
+	verifyFilenameBaseFormat(filenameBase)
+
+	// Get date and slug from filename
+	day := filenameBase[0:2]
+	month := filenameBase[3:5]
+	year := filenameBase[6:10]
+	date, err := time.Parse("02-01-2006", day+"-"+month+"-"+year)
+	if err != nil {
+		log.Fatal(err)
+	}
+	slug := filenameBase[11:]
+
+	// Get body from file
+	mdfile, err := os.Open("./markdown/" + filename)
```

Comment:
I guess this func should respect `DefaultMarkdownPath`, too. Accept path as param?

Reply:
@freetonik  Yep, you are right. This place should be refactored during the #10 implementation